sandbox-backend/src/main/java/com/exadel/sandbox/team5/entity/Country.java

```diff
@@ -18,4 +17,7 @@
 
     @Column(name = "name")
     private String name;
+
+    @OneToMany(mappedBy = "country")
+    private Set<City> cities;
```

Comment:
This field looks like as redundant. You will not save addresses by Country entity or load all by it. For that we will use separate service/repository method